src/main/scala/is/hail/variant/HTSGenotypeView.scala

```diff
@@ -6,7 +6,7 @@ import is.hail.utils._
 
 object HTSGenotypeView {
   def apply(rowSignature: TStruct): HTSGenotypeView = {
-    rowSignature.fields(2).typ.asInstanceOf[TArray].elementType match {
+    rowSignature.fields(3).typ.asInstanceOf[TArray].elementType match {
```

Comment:
I'm getting tired, but why did this change?

Reply:
now it's PK, K, Annotations, GS